public_html/wp-content/themes/pattern-directory/functions.php

```diff
@@ -178,6 +178,7 @@ function pre_get_posts( $query ) {
  */
 function add_rewrite() {
 	add_rewrite_rule( '^my-patterns/[^/]+/?$', 'index.php?pagename=my-patterns', 'top' );
+	add_rewrite_rule( '^favorites/.+/?$', 'index.php?pagename=favorites', 'top' );
```

Comment:
I would expect this URL to include the `my-` convention as well.....`my-favorites`

Reply:
I was on the fence about it, I'll switch it back to `my-favorites`.